modules/core/src/read-file/read-file.js

```diff
@@ -9,7 +9,7 @@ import {
 } from './read-file-browser';
 
 // read-file-node is excluded from build under browser so don't do indivdual imports
-import node from './read-file-node';
+import * as node from './read-file-node';
```

Comment:
Ha. Wonder why the test cases didn't catch this? That is very concerning!

Reply:
This apparently is ok in es6 but breaks when testing transpiled es5. We have not had any dist test in this repo.